src/main/java/ui/UI.java

```diff
@@ -195,7 +186,7 @@ private void initPreApplicationState() {
                 logger.error(throwable.getMessage(), throwable));
 
         TestController.setUI(this, getParameters());
-        prefs = new Preferences(TestController.isTestMode());
+        prefs = new Preferences(TestController.getTestConfigFile(false));
```

Comment:
This looks confusing. Can we have `TestController.createRealPreference()` or `TestController.createApplicationPreference()` (can have a better name) just like for test preference?